hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -632,6 +632,33 @@ public FileSystemViewStorageConfig getClientSpecifiedViewStorageConfig() {
     return clientSpecifiedViewStorageConfig;
   }
 
+  /**
+   * Commit call back configs.
+   */
+  public boolean enableWriteCommitCallback() {
+    return Boolean.parseBoolean(props.getProperty(HoodieWriteCommitCallbackConfig.CALLBACK_ON));
+  }
+
+  public String getCallbackType() {
+    return props.getProperty(HoodieWriteCommitCallbackConfig.CALLBACK_TYPE_PROP);
+  }
+
+  public String getCallbackClass() {
+    return props.getProperty(HoodieWriteCommitCallbackConfig.CALLBACK_CLASS_PROP);
+  }
+
+  public String getCallbackRestUrl() {
+    return props.getProperty(HoodieWriteCommitCallbackConfig.CALLBACK_HTTP_URL_PROP);
+  }
+
+  public int getCallbackRestTimeout() {
+    return Integer.parseInt(props.getProperty(HoodieWriteCommitCallbackConfig.CALLBACK_TIMEOUT_SECONDS));
+  }
+
+  public String getCallbackRestApiKey() {
+    return props.getProperty(HoodieWriteCommitCallbackConfig.CALLBACK_API_KEY);
+  }
```

Comment:
should we move these methods into `HoodieHttpWriteCommitCallback.java`, I am worry about the explode of other callback methods in `HoodieWriteConfig`

Reply:
> should we move these methods into `HoodieHttpWriteCommitCallback.java`, I am worry about the explode of other callback methods in `HoodieWriteConfig`

done